app/controllers/admin/api/cms/files_controller.rb

```diff
@@ -11,9 +11,12 @@ class Admin::Api::CMS::FilesController < Admin::Api::CMS::BaseController
 
   MAX_PER_PAGE = 100
   DEFAULT_PER_PAGE = 20
+  ALLOWED_PARAMS = %i[section_id path attachment downloadable tag_list].freeze
 
   before_action :find_file, only: [:show, :edit, :update, :destroy]
 
+  wrap_parameters :file, include: ALLOWED_PARAMS
```

Comment:
Just to be sure: This is in here so whoever calls this endpoint doesn't have to wrap the body on a `"file":` key, correct? Also, if the body comes wrapped, this will be ignored, right?

Reply:
Exactly